feature/dashboard/src/main/java/com/simprints/feature/dashboard/settings/troubleshooting/adapter/TroubleshootingItemViewData.kt

```diff
@@ -0,0 +1,8 @@
+package com.simprints.feature.dashboard.settings.troubleshooting.adapter
+
+data class TroubleshootingItemViewData(
+    val title: String,
+    val subtitle: String = "",
+    val body: String = "",
+    val navigationId: String? = null,
```

Comment:
`navigationId` is always being referenced as `scopeId` in other callers. Don't we want to rename it to `scopeId`/`eventScopeId` for consistency?

Reply:
The view data is generic on purpose so that we can reuse the same layouts and adapter combos in multiple screens.